src/app/caesar-portal/group-area/group-area.component.html

```diff
@@ -0,0 +1,46 @@
+<div class="col-md-7 group-wrapper">
+    <div class="row">
+        <div class="col-md-6">
+            <h1>Dnipro</h1>
+        </div>
+        <div class="col-md-6 text-right">
+            <h1>Dp-094 MQC</h1>
+        </div>
+    </div>
+    <div class="row group-area">
+        <div class="row">
+            <div class="col-md-2">
+                <a>
+                    <span class="glyphicon glyphicon-cog" aria-hidden="true"></span>
+                </a>
+            </div>
+            <div class="col-md-10 text-right">
+                <div class="btn-group" role="group" aria-label="...">
+                    <button (click)="onTabChange('info')" type="button" class="btn btn-default" [ngClass]="{active: activeTab === 'info'}">
```

Comment:
why are you doing this as tabs? a week ago we agreed that all of this will be separate routes, not tabs